apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.spec.js

```diff
@@ -215,6 +220,14 @@ describe('notifyResidentsAboutNewsItem', () => {
                 expect(message1).toBeDefined()
                 expect(message1.id).toMatch(UUID_RE)
 
+                // Check content for notifications
+                expect(message1).toEqual(expect.objectContaining({
+                    meta: expect.objectContaining({
+                        title: expect.not.stringMatching(THREE_DOT_FOR_TEST_REGEXP),
+                        body: expect.not.stringMatching(THREE_DOT_FOR_TEST_REGEXP),
+                    }),
+                }))
+
```

Comment:
Maybe it would be better to declare the variables `MESSAGE_TITLE_MAX_LEN` and `MESSAGE_BODY_MAX_LEN` in this file with tests too?
Thus we can't update the text length without changing the tests

Reply:
And in this case you don't have to add this strange check

Reply:
I'd have rewritten the check too.. You need to chekck pattern like this [a-z]*... and maybe use startsWith, it will check that 

1. You actually are sending some content in message
2. Message is exactly what you'd expect (original title and description starts with shortened message w/o ...)